arch/varnish.install

```diff
@@ -0,0 +1,12 @@
+# remove localstatedir -- compiled VCL and slab files will still be here.
+post_remove() {
+  rm -rf var/lib/varnish
+}
+
+post_upgrade() {
+  if [[ $(vercmp 6.0.0-1 "$2") -eq 1 ]]; then
+    echo ":: You must update your VCL to be compatible with varnish 6. See the"
+    echo "   following upgrade guide to port your configuration:"
+    echo "   http://varnish-cache.org/docs/6.2/whats-new/upgrading-6.2.html"
+  fi
+}
```

Comment:
Strange 6.0.0 comparison with 6.2 docs.

Reply:
I'm getting rid of it altogether, people ought to be aware of it by now, right?

RIGHT??!?!?